docs/pages/blog/lab-date-pickers-to-mui-x.md

```diff
@@ -51,8 +51,8 @@ And by offering an even more advanced component package through our commercial l
 
 The date and time picker components are now available in two packages:
 
-- `@mui/x-date-pickers` which contains all the MIT licensed components
-- `@mui/x-date-pickers-pro` which contains all the MIT and commercially licensed components
+- `@mui/x-date-pickers` has a [MIT license](https://unpkg.com/browse/@mui/x-date-pickers/LICENSE), it contains all the open-source components
+- `@mui/x-date-pickers-pro` has a [commercial license](https://unpkg.com/browse/@mui/x-date-pickers-pro/LICENSE), it contains all the MIT and commercially licensed components
```

Comment:
This is meant to train the audience to understand the notion of a license, it's **this file** that matters.